<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1526</width>
    <height>1256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MouseTrackingLabel" name="ui_label_view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="ui_slider_t">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1050</y>
      <width>1021</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ui_slider_shiftx">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>0</y>
      <width>20</width>
      <height>1021</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="ui_slider_shifty">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>0</y>
      <width>20</width>
      <height>1021</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="ui_group_mask">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>240</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Mask</string>
    </property>
    <widget class="QComboBox" name="ui_combo_masks">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_mask_delete">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_mask_add">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_mask_delete_all">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete All</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ui_label_x">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>1020</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui_label_y">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>1030</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui_label_signal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1090</y>
      <width>1021</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="ui_group_data">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>0</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QLineEdit" name="ui_edit_vpath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>../../data/neurofinder/00.00/data/00.00.h5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_vselect">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_vload">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_vunload">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Unload</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_mpath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Masks:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_mpath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>../../data/neurofinder/00.00/data/regions.json</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_vpath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Video:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_msave">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_mload">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_mselect">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_max_frames">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Frames</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_max_frames">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ui_group_control">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>440</y>
      <width>431</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QLabel" name="ui_label_pos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>[PosX, PosY]</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_value">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_value2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_prev_t">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_next_t">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_t">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_set_t">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_zoom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
    <widget class="QSlider" name="ui_slider_zoom">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_pre_process">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre-Process Data</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="ui_progress">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>32</y>
       <width>118</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_combo_mode">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_mode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_intclip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Intensity Clipping</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_int_low">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_int_high">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_int_clip">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_ppath">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_psave">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_pload">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_ppath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
    <widget class="QSlider" name="ui_slider_mask_alpha">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>70</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ui_label_t">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>1050</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ui_group_mask_analysis">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>840</y>
      <width>421</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Mask Analysis</string>
    </property>
    <widget class="QPushButton" name="ui_button_measure_run">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Measure Mask</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_histogram">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>381</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_measure_clear">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_safety_margin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Safety Margin</string>
     </property>
    </widget>
    <widget class="QSlider" name="ui_slider_safety_margin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_safety_margin_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Background Margin</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
    </widget>
    <widget class="QSlider" name="ui_slider_background_margin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TickPosition::TicksBelow</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseTrackingLabel</class>
   <extends>QLabel</extends>
   <header>src/widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
